android/app/src/main/java/app/organicmaps/bookmarks/BookmarksListFragment.java

```diff
@@ -810,7 +810,7 @@ public void onPreparedFileForSharing(@NonNull BookmarkSharingResult result)
   public void onActivityResult(int requestCode, int resultCode, Intent data)
   {
     super.onActivityResult(requestCode, resultCode, data);
-    getAdapter().notifyDataSetChanged();
+    getBookmarkListAdapter().notifyDataSetChanged();
```

Comment:
This looks misleader, but I can see that the same approach is used across the code.

Reply:
Yeah, I didn't want to change the pattern to fix the bug.